deeptrack/models/gans/cgan.py

```diff
@@ -0,0 +1,121 @@
+import tensorflow as tf
+from tensorflow.python.keras.utils.generic_utils import default
+from .utils import as_KerasModel
+
+layers = tf.keras.layers
+
+
+@as_KerasModel
+class CGAN(tf.keras.Model):
```

Comment:
Add documentation: Small description of the network and what it does, followed by a description of each argument

Reply:
:white_check_mark: Documentation added